### PR TITLE
169/save user session in cookie and test user webservice

### LINKED ISSUE #169: save user session in cookie and test user webservice

### PR DESCRIPTION
**Pull request by iLeonelPerea**
closes #169 

changes proposed in the pull request:

- Don't start drakov service
- Save user_session in cookie
- Modify DEVELOPMENT host value

@thetonymaster

### CODE REVIEW

File: src/containers/ProvisionerForm.js [hunk 1]
```diff
@@ -16,10 +18,14 @@ const provisionFormOptionsApi = require("../api/provisionFormOptionsApi");
 
 export class ServiceForm extends Component {
   componentWillMount() {
-    if (!this.props.userAppState.get('user_session'))
+    if (!cookie.load('user_session'))
```
Comment (thetonymaster) [inline src/containers/ProvisionerForm.js]: What happens if the cookie does not exists?

File: src/containers/UserLogin.js [hunk 2]
```diff
@@ -3,12 +3,13 @@ import { connect } from 'react-redux';
...
 import { browserHistory } from 'react-router';
 import { fromJS } from 'immutable';
+import cookie from 'react-cookie';
 import * as actions from '../actions/userActions';
 import Login from "../components/userAuth/Login";
 
 export class UserLogin extends Component {
   componentWillMount() {
-    if (this.props.userAppState.get('user_session'))
+    if (cookie.load('user_session'))
```
Comment (thetonymaster) [inline src/containers/UserLogin.js]: Same here

**Comment by thetonymaster** [comment]
Don't or won't start?

File: src/containers/ProvisionerForm.js (hunk 1 above)
Comment (iLeonelPerea) [inline src/containers/ProvisionerForm.js]: the condition redirect to /login when cookie does not exist..

File: src/containers/UserLogin.js (hunk 2 above)
Comment (iLeonelPerea) [inline src/containers/UserLogin.js]: When the cookie exist redirect to /

**Comment by thetonymaster** [comment]
LGTM

**Comment by thetonymaster** [comment]
Add later a new command that also runs drakov. Just to test locally.

**Pull request merged**